actions/scripts/create_run_hls_script.sh

```diff
@@ -114,6 +114,6 @@ config_interface -m_axi_addr64=true
 #config_rtl -reset all -reset_level low
 
 csynth_design
-export_design -format ip_catalog
+#export_design -format ip_catalog -rtl vhdl
```

Comment:
This one has been commented until the complete integration of those specific HLS IP we are facing with math functions. This will be part of a larger patch.